ql/src/java/org/apache/hadoop/hive/ql/exec/tez/monitoring/RenderStrategy.java

```diff
@@ -161,7 +166,8 @@ public void renderReport(String report) {
       if (hiveServer2InPlaceProgressEnabled) {
         LOGGER.info(report);
       } else {
-        monitor.console.printInfo(report);
+        String time = REPORT_DATE_TIME_FORMATTER.format(LocalDateTime.now());
```

Comment:
LocalDateTime won't be confusing? Just thinking how it will be look like if a user will be EST but the server time will be PST. 
Just try to imagine the output. 

Overall the code is OK. Could you please move the jira in-progress state?

Reply:
I followed the behavior of MapReduce since I actually wrote this patch while I was migrating Hive on MR on Hive on Tez.
- MR: It uses [the default Calendar](https://github.com/apache/hive/blob/31c1658d9884eb4f31b06eaa718dfef8b1d92d22/ql/src/java/org/apache/hadoop/hive/ql/exec/mr/HadoopJobExecHelper.java#L350) which follows [the default time zone](https://docs.oracle.com/javase/8/docs/api/java/util/Calendar.html#getInstance--)

Looking at other usages, it could be kinder to use [HiveConf#getLocalTimeZone](https://github.com/apache/hive/blob/rel/release-4.0.0-alpha-2/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java#L6482-L6485). Let me check and try it.

Reply:
@aturoczy I updated it. I think it is reasonable since it is consistent with other places such as [CURRENT_DATE UDF](https://github.com/apache/hive/blob/rel/release-4.0.0-alpha-2/ql/src/java/org/apache/hadoop/hive/ql/udf/generic/GenericUDFCurrentDate.java#L55).

I have also verified the property takes effect.

By default, the default one, UTC in this case, is used.

```
$ beeline -e 'SELECT sum(price) FROM orders' --hiveconf hive.server2.in.place.progress=false
...
INFO  : 2023-06-16 16:56:55,735	Map 1: -/-	Reducer 2: 0/1	
INFO  : 2023-06-16 16:56:56,240	Map 1: 0/1	Reducer 2: 0/1	
```

The same offset is used when we explicitly give `hive.local.time.zone=UTC`.

```
$ beeline -e 'SELECT sum(price) FROM orders' --hiveconf hive.server2.in.place.progress=false --hiveconf hive.local.time.zone=UTC
...
INFO  : 2023-06-16 16:59:44,985	Map 1: -/-	Reducer 2: 0/1	
INFO  : 2023-06-16 16:59:45,995	Map 1: 0/1	Reducer 2: 0/1	
```

Configuring other time zones, we see a date and time with the offset.

```
$ beeline -e 'SELECT sum(price) FROM orders' --hiveconf hive.server2.in.place.progress=false --hiveconf hive.local.time.zone=America/Los_Angeles
...
INFO  : 2023-06-16 10:02:17,700	Map 1: -/-	Reducer 2: 0/1	
INFO  : 2023-06-16 10:02:18,712	Map 1: 0/1	Reducer 2: 0/1	
...
$ beeline -e 'SELECT sum(price) FROM orders' --hiveconf hive.server2.in.place.progress=false --hiveconf hive.local.time.zone=Asia/Tokyo
...
INFO  : 2023-06-17 02:03:21,041	Map 1: -/-	Reducer 2: 0/1	
INFO  : 2023-06-17 02:03:22,050	Map 1: 0/1	Reducer 2: 0/1	
```